frontend/src/components/program/picasso/PicassoEntry.vue

```diff
@@ -82,11 +83,22 @@ export default {
     editable: { type: Boolean, required: true },
     scheduleEntry: { type: Object, required: true },
     timed: { type: Boolean, required: true },
-    activityName: { type: String, required: true },
-    scheduleEntryRoute: { type: Object, required: true },
   },
   emits: ['startResize', 'finishEdit'],
   computed: {
+    activityName() {
+      if (this.scheduleEntry.tmpEvent) return this.$tc('entity.activity.new')
+
+      if (this.activityLoading) return this.$tc('global.loading')
+
+      return (
+        (this.scheduleEntry.number ? this.scheduleEntry.number + ' ' : '') +
+        (this.scheduleEntry.activity().category().short
```

Comment:
Could you extract the activity, now that it is used this many times?

Reply:
Good point. Fixed in 01b117630c7dffdc790774f23f87ebbe8da58889